interface-elements-for-web/articles/grid/customization-dialog/customization-dialog.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Customization Dialog
+---
+# Customization Dialog
+The grid provides a customization dialog that enables you to perform data shaping operations.
```

Comment:
The customization dialog enables you to perform data shaping operations? OR You can use the customization dialog to perform data shaping operations?